code/modules/mob/living/damage_procs.dm

```diff
@@ -58,11 +58,25 @@
 /mob/living/proc/apply_internal_damage(var/damage = 0, var/organ)
 	return
 
+#define EFFECT_FLAG_LIFE (1>>0)
+#define EFFECT_FLAG_DEFAULT (1>>1)
```

Comment:
pretty sure these bitflag arrows are pointing the wrong way

Reply:
durrrrrp